Source/Eto/Forms/Layout/TableLayout.cs

```diff
@@ -158,6 +224,22 @@ public TableLayout(IEnumerable<TableRow> rows)
 			Initialize();
 		}
 
+		/// <summary>
+		/// Initializes a new instance of the <see cref="Eto.Forms.TableLayout"/> class with the specified rows.
+		/// </summary>
+		/// <param name="yscale">Scale all rows</param>
+		/// <param name="rows">Rows to populate the table.</param>
+		public TableLayout(bool yscale, params TableRow[] rows)
+		{
```

Comment:
very usefull, I hope `yscale` is good name (`scale` could confuse as it does not scale cells)